econml/dr/_drlearner.py

```diff
@@ -1523,7 +1523,7 @@ def n_crossfit_splits(self, value):
 
     @property
     def criterion(self):
-        return self.criterion
+        return "mse"
```

Comment:
Seems a bit weird that we allow `est.criterion = 'deprecated'` but not `est.criterion = 'mse'`, and yet we return `"mse"` here...

If this is identical to sklearn's approach then I guess we can stick with it, but seems unintuitive to me.

Reply:
Not exactly the same approach, they have a decorator for deprecated args and the deprecated args are not defined in init. Or can I just remove these deprecated args? since it should be removed for next release anyway?

Reply:
@kbattocchi @vsyrgkanis Any final conclusion on this? Then I could make it ready to merge.

Reply:
How hard would it be to mimic sklearn's approach?  I think it would be best to either do that or remove the deprecated args as you suggested.

Reply:
I removed all the deprecated args, also for other classes, especially clean up the n_splits/n_crossfit_splits. It's not so clear to me how to mirror sklearn, they seem just add a warning, might worth create an issue to explore how to handle that systematically.